src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperationHandler;
+import core.basesyntax.service.FruitService;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private static FruitService fruitService;
+    private static final String expected = "fruit,quantity" + System.lineSeparator()
```

Comment:
move this to the method, where you need it. Now it looks a bit confusing that you have several tests and initialize expected here